plugins/action/azure_rm_arcssh.py

```diff
@@ -10,6 +10,7 @@
 from ansible.errors import AnsibleActionFail
 from ansible.plugins.action import ActionBase
 from ansible.utils.display import Display
+from azure.core.exceptions import ClientAuthenticationError, ResourceExistsError, ResourceNotFoundError
```

Comment:
Due to the ansible-test in isolate environment , please import the relevant azure modules as follows. Thank you!
```suggestion
try:
    from azure.core.exceptions import ClientAuthenticationError, ResourceExistsError, ResourceNotFoundError
except ImportError:
    pass
```